themes/default/layouts/partials/banner.html

```diff
@@ -1,6 +1,6 @@
-{{ $summary := "Learn about the future of Cloud Engineering at PulumiUP." }}
-{{ $url := relref . "/pulumi-up" }}
-{{ $label := "Save Your Spot"}}
+{{ $summary := "Pulumi 3.0 has launched and with it comes a whole new set of capabilitie for you to harness the cloud like never before! 🚀" }}
```

Comment:
shorter will prolly work better here, how about..

`[Pulumi 3.0](blog) has launched :rocket:! Harness the cloud like never before!` or similar?